packages/components/src/Cluster/types.ts

```diff
@@ -17,15 +26,40 @@ export interface ClusterProps {
   /**
    * The amount of space between the cluster elements. Semantic tokens are available.
    */
-  readonly space?: Spaces | (string & NonNullable<unknown>);
+  readonly gap?: Spaces | (string & NonNullable<unknown>);
 
   /**
    * The breakpoint below which the cluster will collapse.
    */
-  readonly collapseBelow?: typeof Breakpoints | (string & NonNullable<unknown>);
+  readonly collapseBelow?: keyof typeof AtlantisBreakpoints;
+
+  /**
+   * Force the cluster to collapse. Use this when our breakpoints are not enough control.
+   */
+  readonly collapsed?: boolean;
 
   /**
    * Enabling this prevents the cluster from taking 100% of the width of the parent and instead flows with the content.
    */
   autoWidth?: boolean;
+
+  /**
+   * The HTML tag to render the cluster as.
+   */
+  as?: CommonAllowedElements;
+
+  /** **Use at your own risk:** Custom class names for specific elements. This should only be used as a
+   * **last resort**. Using this may result in unexpected side effects.
+   * More information in the [Customizing components Guide](https://atlantis.getjobber.com/guides/customizing-components).
+   */
+  readonly UNSAFE_className?: {
+    container?: string;
```

Comment:
I had a question on this? I realize its consistent, but requires an object to be created for elements with only 1 component like this? Same comment about styles.

Reply:
I think it could become a clear rule that if it has more than one override, then it's an object; otherwise, it's a string.

Reply:
Myself, @jdeichert & @ZakaryH have discussed this before when adding `UNSAFE_` to atom components that are also just a container. Jacob's suggestion was that we used a keyed map to make sure we're handling future scenarios/future proofing (even if it seems unlikely any other sub element will be added to a component). 
I did initially have the same thoughts, and we _could_ have just a string for className and an object for `_styles` if people feel strongly. I tend to lean into consistency at this point.

Reply:
That's fair. I think it probably speaks to components needing more composition so that the overrides could be targetted, but another way to approach that with the current structure would be to actually give `data-slot="<componentName>"` which could then be targetted by the CSS through data attribute selectors rather than needing to be the ones responsible for plumbing classnames through.

Forgive the tailwind nuances here that they're hiding behind Components, but he presents an interesting approach to controlling styles from above using data attributes. https://www.youtube.com/watch?v=MrzrSFbxW7M

Reply:
Wonder if we just leverage what Braid does? https://github.com/seek-oss/braid-design-system/blob/0de668c186fd972a197160b799b980243ef58dc7/packages/braid-design-system/src/lib/components/private/buildDataAttributes.ts

Then you don't need to re-type data?